spec/lib/mvi/service_spec.rb

```diff
@@ -262,21 +258,6 @@
       end
     end
 
-    context 'when a MVI failure response is returned' do
-      it 'should raise a request failure error', :aggregate_failures do
```

Comment:
This is a duplicate of the preceding spec.  
It is built with the same request XML, so the results will be exactly the same.